changelogs/fragments/364-vmware_vmkernel-replciation-fix.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - vmware_vmkernel - fixed issue were Repl and ReplNFC services were not being identified as enabled on a vmk interface (https://github.com/ansible-collections/vmware/issues/362).
```

Comment:
were -> where (the first one)

Reply:
Fixed now

Reply:
Great, thank you for your reactivity :-)